api/websocket.go

```diff
@@ -2,12 +2,14 @@ package api
 
 import (
 	"context"
+	"math"
 	"net/http"
 	"sync"
 	"time"
 
 	"github.com/gorilla/websocket"
 	"go.uber.org/zap"
+	"golang.org/x/time/rate"
```

Comment:
https://github.com/uber-go/ratelimit

Reply:
`golang.org/x/time/rate` already meets our needs, i think no need to import a third-party library